blueflood-core/src/main/java/com/rackspacecloud/blueflood/utils/GlobPattern.java

```diff
@@ -0,0 +1,169 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+/**
+ * Copied from http://svn.apache.org/repos/asf/hadoop/common/branches/branch-1/src/core/org/apache/hadoop/fs/GlobPattern.java
+ */
```

Comment:
I don't know svn, but if it's possible to specify exactly which commit/branch this came from that would be good.